src/Bicep.LangServer/Completions/ModuleReferenceCompletionProvider.cs

```diff
@@ -27,14 +30,26 @@ public class ModuleReferenceCompletionProvider : IModuleReferenceCompletionProvi
         private readonly ISettingsProvider settingsProvider;
         private readonly ITelemetryProvider telemetryProvider;
 
-        private static readonly ImmutableDictionary<string, (string, CompletionPriority)> BicepRegistryAndTemplateSpecShemaCompletionLabelsWithDetails = new Dictionary<string, (string, CompletionPriority)>()
+        private enum ModuleCompletionPriority
         {
-            {"br:", ("Bicep registry schema name", CompletionPriority.VeryHigh) },
-            {"br/", ("Bicep registry schema name", CompletionPriority.High) },
-            {"ts:", ("Template spec schema name", CompletionPriority.VeryHigh) }
+            Alias = 0,     // br/[alias], ts/[alias]
+            Default = 1,
+            FullPath = 2,  // br:, ts:
+        }
+
+        private const string ModuleFullPathCompletionLabel = "Bicep registry";
+        private const string ModuleAliasCompletionLabel = "Bicep registry (alias)";
+        private const string TemplateSpecFullPathCompletionLabel = "Template spec";
+        private const string TemplateSpecAliasCompletionLabel = "Template spec (alias)";
+
+        private static readonly ImmutableDictionary<string, (string, ModuleCompletionPriority)> BicepRegistryAndTemplateSpecShemaCompletionLabelsWithDetails = new Dictionary<string, (string, ModuleCompletionPriority)>()
+        {
+            {"br:", (ModuleFullPathCompletionLabel, ModuleCompletionPriority.FullPath) },
+            {"br/", (ModuleAliasCompletionLabel, ModuleCompletionPriority.Alias) },
+            {"ts:", (TemplateSpecFullPathCompletionLabel, ModuleCompletionPriority.FullPath) }
         }.ToImmutableDictionary();
 
-        private static readonly Regex ModuleRegistryWithoutAlias = new Regex(@"'br:(?<registry>(.*?))/'?$", RegexOptions.Compiled | RegexOptions.ExplicitCapture | RegexOptions.IgnoreCase);
+        private static readonly Regex ModuleRegistryWithoutAlias = new Regex(@"'br:(?<regxistry>(.*?))/'?$", RegexOptions.Compiled | RegexOptions.ExplicitCapture | RegexOptions.IgnoreCase);
```

Comment:
typo?

Reply:
That explains the bug I was trying to track down...